packages/examples/dashboard/src/components/Dashboard/Dashboard.tsx

```diff
@@ -1,54 +1,46 @@
-import React from 'react'
 import {
   AccessTokenProvider,
-  ThemeProvider,
-  useTheme,
+  DefaultThemes,
   FilterProvider,
+  RelativeTimeRange,
   SimpleFilter,
-  RelativeTimeRange
+  ThemeProvider,
+  useTheme
 } from '@propeldata/ui-kit'
-import {
-  TimeSeriesStaticTest,
-  TimeSeriesConnectedTest,
-  LeaderboardStaticTest,
-  LeaderboardConnectedTest,
-  CounterStaticTest,
-  CounterConnectedTest,
-  PieChartStaticTest,
-  PieChartConnectedTest
-} from 'components'
-
-const { REACT_APP_CLIENT_ID, REACT_APP_CLIENT_SECRET } = process.env
-
-async function fetchToken() {
-  const response = await fetch('https://auth.us-east-2.propeldata.com/oauth2/token', {
-    method: 'post',
-    body: `grant_type=client_credentials&client_id=${REACT_APP_CLIENT_ID}&client_secret=${REACT_APP_CLIENT_SECRET}`,
-    headers: { 'Content-Type': 'application/x-www-form-urlencoded' }
-  })
-  const { access_token } = await response.json()
-
-  return access_token
-}
+import React from 'react'
+import { DashboardCommonProps } from '../../shared.types'
+import { CounterConnected } from '../CounterConnected'
+import { CounterStatic } from '../CounterStatic'
+import { LeaderboardConnected } from '../LeaderboardConnected'
+import { LeaderboardStatic } from '../LeaderboardStatic'
+import { PieChartConnected } from '../PieChartConnected'
+import { PieChartStatic } from '../PieChartStatic'
+import { TimeSeriesConnected } from '../TimeSeriesConnected'
+import { TimeSeriesStatic } from '../TimeSeriesStatic'
 
 const GlobalStyles = () => {
   const theme = useTheme()
   if (document && theme) {
-    document.body.style.setProperty('--bg-color', theme.bgSecondary)
+    document.body.style.setProperty('--bg-color', theme.bgSecondary as string)
   }
   return null
 }
 
-export default function App() {
-  const [theme, setTheme] = React.useState('lightTheme')
+interface DashboardProps extends DashboardCommonProps {
+  fetchToken: () => Promise<string>
+  reactVersion: string
+}
+
+export const Dashboard = ({ fetchToken, reactVersion, envs }: DashboardProps) => {
+  const [theme, setTheme] = React.useState<DefaultThemes>('lightTheme')
   return (
     <AccessTokenProvider fetchToken={fetchToken}>
       <FilterProvider>
         <ThemeProvider baseTheme={theme}>
           <GlobalStyles />
           <main style={{ color: 'var(--propel-text-secondary)', backgroundColor: 'var(--propel-bg-secondary)' }}>
             <h1 className="px-6 py-3 text-3xl">
-              React 18 Testing App
+              React {reactVersion} Testing App
```

Comment:
Maybe it would be more accurate to use `React.version` instead of having to pass the version as a prop? That way we can make 100% sure the version running is the expected one